src/test/end-to-end/codecept.conf.js

```diff
@@ -8,7 +8,7 @@ exports.config = {
             url: config.TestUrl,
             waitForTimeout: 40000,
             getPageTimeout: 40000,
-            // waitForAction: 1000,
+            waitForAction: 4000,
```

Comment:
The tests will take longer to run if we use this wait.

Reply:
Yes I appreciate this...the issues is sometime the actions is not properly completed if this timelimit is kept low...However I will discuss this with the team before a merge to master...